modules/015-admission-policy-engine/hooks/bootstrap_handler.go

```diff
@@ -62,8 +85,29 @@ func handleGatekeeperBootstrap(input *go_hook.HookInput) error {
 		}
 	}
 
-	deploymentReady := snap[0].(bool)
-	input.Values.Set("admissionPolicyEngine.internal.bootstrapped", deploymentReady)
+	// check if deployment is ready
+	bootstrapped := snap[0].(bool)
```

Comment:
This will flap on the deployment restart

Reply:
updated the bootstrap handler so it checks only required templates